src/pages/Transactions/index.tsx

```diff
@@ -156,6 +157,21 @@ const _TransactionsPage = ({
       title={formatMessage({ defaultMessage: 'Transactions' })}
     >
       <Paper className="mt20">
+        {depositKeys.find(
+          k => k.transactionStatus === TransactionStatus.LEDGER_ERROR
+        ) && (
+          <Alert variant="error" className="mb20">
+            <FormattedMessage
+              defaultMessage="There was a problem trying to sign with your Ledger device. Please verify your Ledger
+              device has both {blindSigning} and {debugData} enabled. These can be accessed by selecting the Ethereum
+              application and then entering the Settings menu."
+              values={{
+                blindSigning: <b>Blind Signing</b>,
+                debugData: <b>Debug Data</b>,
```

Comment:
@valefar-on-discord Double checking here, do we want these to be translated? I'm not deeply familiar with any non-English Ledger interface. If we want these terms to be translated, we'll need to  wrap them in their own `FormattedMessage` component.

Reply:
That's a great question but I don't believe they are translated.

If you take a look at the [ledger documentation in a different language](https://support.ledger.com/hc/de/articles/4405481324433-Aktivieren-der-Blindsignatur-in-der-Ethereum-ETH-App?docs=true), it will show "Blind Signing"

Reply:
I'm going to update the instructions based on this little discovery.

Reply:
Updated to extract hard-coded words and used `<strong>`